### DISCUSSION TITLE
GitHub Bot/App

### DISCUSSION THREAD

**Discussion by 0xAndrewBlack** [community/community#33179] [New to GitHub]
Hello

I want to make a GitHub bot that listens to specific events, I did some research and found ProBot but Its weird syntax and pattern threw me off are there frameworks or example apps similar to discord.js and such?

TLDR; I want a small codebase to listen to events like Discord.JS and do stuff with the event and its data.

Thanks in advance.

**Comment by DSDmark** [comment]
For Automatically add and move Issues and Pull Requests on a Project board.
You can use git action like this.
https://github.com/apps/project-bot

Yes, there are no such type of well documentation about GitHub bot, but you can try this maybe it helps you
https://github-bot-tutorial.readthedocs.io/en/latest/why-github-bots.html#what-are-github-bots
